src/components/ServiceArea.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+
+const ServiceArea = () => {
+  return (
+    <div className='bg-gray-100'>
+      <div className='px-6 pt-10'>
+        <h2 className='text-center mb-3 text-5xl md:text-6xl'>Areas We Serve</h2>
+        <hr className='mx-10 bg-black' />
+        <p className='text-xl my-10 md:text-center'>We are proudly located in Detroit, Michigan. We service the <span className='font-bold'>Detroit, Highland Park, Hamtramck,</span> and <span className='font-bold'>metro<br /> Detroit </span>area. Unsure if we serve your location? Give us a call!
```

Comment:
question: Why are we using a `<br />` tag in here? On different screen sizes it inserts line breaks which look bad. Remember, we are not coding just for one screen size, but all screen sizes, so if you are unsure whether or not there must be a line break between some text, it is best to ask the designer how the design should look at different screen sizes.